src/parsers/manifest/local/parse_local_manifest.ts

```diff
@@ -76,10 +71,10 @@ function parsePeriod(
   period : ILocalPeriod,
   ctxt : { periodIdGenerator : () => string; /* Generate next Period's id */
            /** If true, the local Manifest has been fully downloaded. */
-           isFinished : boolean; }
+           localManifest : ILocalManifest; }
```

Comment:
Giving the whole `ILocalManifest` is kind of ugly no (in that we don't know what is actually used)? Why not just giving the few parameters needed?